app/views/questions_index.erb

```diff
@@ -0,0 +1,4 @@
+<div class='container'>
+  <% @voteable = @question %>
+  <%= erb: 'votes/_vote_form' %>
```

Comment:
@jdollete This isn't the most restful filename and I already created a questions/index page for us-- I'm going to go ahead and merge this pull request but let's paste the contents of the file into the questions/index.erb file, and then delete this one.